theano/__init__.py

```diff
@@ -195,11 +191,11 @@ def sparse_grad(var):
     ret = var.owner.op.__class__(sparse_grad=True)(*var.owner.inputs)
     return ret
 
-# This cannot be done in tensor/__init__.py due to a circular dependency -- randomstreams
-# depends on raw_random which depends on tensor.  As a work-around, we import RandomStreams
-# here and inject an instance in tensor.
-from theano.tensor.randomstreams import RandomStreams
+# This cannot be done in tensor/__init__.py due to a circular
+# dependency -- shared_randomstreams depends on raw_random which
+# depends on tensor.  As a work-around, we import RandomStreams here
+# and inject an instance in tensor.
+from theano.tensor.shared_randomstreams import RandomStreams
```

Comment:
Are we sure we want to do that? replace theano.tensor.RandomStreams to the new RamdonStream? I think I would not do that for now.

We have a plan to make a meta random interface, that would take an algorithm as a parameter. I think I would keep that old interface available for this.